website_url_sitemap_exclusion/models/website_forbidden_url.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2023 Simone Rubino - TAKOBI
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class WebsiteForbiddenUrls(models.Model):
+    _name = "website.forbidden.url"
```

Comment:
Maybe matter of taste but I would use "blacklist(ed)" instead of "forbidden".

Reply:
maybe good for the roadmap? :wink:

Reply:
👍🏻 added to the roadmap